README.md

```diff
@@ -0,0 +1,18 @@
+![](https://github.com/karam084/My_Portfolio_WebSite-)
+
+# My Portfolio Web Site
+
+- The project include on (index.html and style.css)
+- Contents of the project for my previous work details
+
+## Built With
+
+- html and css
+- Visual Studio Code
+
+## Authors
+
+=======
+👤 **Author1**
+
+- GitHub: [@karam084](https://github.com/karam084/Hello_Microverse1)
```

Comment:
- [ ] Please your Readme file is incomplete, follow this Microverse [template](https://github.com/microverseinc/readme-template), personalize it and fill out the available information.